Package.swift

```diff
@@ -18,17 +18,8 @@ let package = Package(
             name: "CrashReporter",
             path: "",
             exclude: [
-                "Source/dwarf_opstream.hpp",
-                "Source/dwarf_stack.hpp",
-                "Source/PLCrashAsyncDwarfCFAState.hpp",
-                "Source/PLCrashAsyncDwarfCIE.hpp",
-                "Source/PLCrashAsyncDwarfEncoding.hpp",
-                "Source/PLCrashAsyncDwarfExpression.hpp",
-                "Source/PLCrashAsyncDwarfFDE.hpp",
-                "Source/PLCrashAsyncDwarfPrimitives.hpp",
-                "Source/PLCrashAsyncLinkedList.hpp",
```

Comment:
These need to be included or the build doesn't work. `swift build` doesnt fail here because `exclude` headers are still available to include paths etc during a build AFAICT. However, deleting the file and trying to build shows the expected failure. 

These may have been excluded because SPM/Swift did not support this previously but as of Swift 5.9 it is supported: https://www.swift.org/documentation/cxx-interop/